packages/suite/src/utils/suite/homescreen.ts

```diff
@@ -228,8 +228,17 @@ export const imagePathToHex = async (imagePath: string, deviceModel: DeviceModel
 export const isHomescreenSupportedOnDevice = (device: TrezorDevice) => {
     const deviceModel = getDeviceModel(device);
 
+    if (deviceModel !== DeviceModel.TT) return true;
+
+    // not sure if this was ever released to production, maybe could be safely removed
+    // @ts-expect-error (reported like this in some previous version)
+    if (deviceModel === DeviceModel.TT && device.features?.homescreen_format === 'Jpeg240x240')
+        return true;
+
     return (
-        deviceModel !== DeviceModel.TT ||
-        (deviceModel === DeviceModel.TT && device.features?.homescreen_format === 'Jpeg240x240')
+        deviceModel === DeviceModel.TT &&
+        device.features?.homescreen_format === 'Jpeg' &&
```

Comment:
Probably it will be enough to check, that `homescreen_format` is defined. So you can remove the condition above and checking width and height

Reply:
something like this? [259b9b9](https://github.com/trezor/trezor-suite/pull/8845/commits/259b9b9f7979eba19ee6d62b28d569a5b2a21215)